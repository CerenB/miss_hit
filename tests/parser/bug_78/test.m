% (c) Copyright 2020 Florian Schanda

disp f)oo % bar  % 'f)oo'
disp f)oo bar    % 'f)oo bar'
disp f(o,o)      % 'f(o,o)'
disp f(o, oo)    % 'f(o, oo)'
disp f(o[ b      % 'f(o[ b'
disp f] oo       % 'f] oo'
disp f]f[[ bar   % 'f]f[[ bar'
disp foo(%bar    % 'foo('
disp ]foo        % ']foo'
disp }foo        % '}foo'
d2 foo )foo      % 'foo' ')foo'
