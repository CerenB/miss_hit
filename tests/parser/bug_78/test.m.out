----------------------------------------------------------------------
--  Parse tree for test.m
Root: Script_File
  Statements: Sequence_Of_Statements
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'f)oo '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'f)oo bar    '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'f(o,o)'
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'f(o, oo)'
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'f(o[ b      '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'f] oo       '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'f]f[[ bar   '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'foo('
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal 'foo( '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal ']foo        '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <disp>
        Arguments: Char_Array_Literal '}foo        '
    Statements: Naked_Expression_Statement
      Expression: Command form Function_Call
        Name: Identifier <d2>
        Arguments: Char_Array_Literal 'foo'
        Arguments: Char_Array_Literal ')foo      '
----------------------------------------------------------------------
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
